qase-playwright/changelog.md

```diff
@@ -1,3 +1,17 @@
+# playwright-qase-reporter@2.0.3
+
+## What's new
+
+Added new annotation `qase.suite()`.
+Tests marked with it will be grouped in the Qase report by the specified suite name.
```

Comment:
Not grouped, but "reported to the specified suite". 

+ describe how it interacts with `QASE_ROOT_SUITE`

+ what happens if tests are annotated with IDs: who wins? Will the results stay or move?

Reply:
Added more information.

If a client uses APIv1 and specify QaseID then the reporter will ignore the suites. 
If a client uses APIv2 and specify QaseID then the reporter will upload results with new suites.

Reply:
Thanks!